cmd/http/tracer.go

```diff
@@ -62,9 +62,9 @@ func (s ProbabilisticSampler) Description() string {
 }
 
 func NewProbabilisticSampler(probability float64) *ProbabilisticSampler {
-	const max = 1000.0
-	boundary := max * probability
-	return &ProbabilisticSampler{max, int(boundary)}
+	const MAX = 1000.0
```

Comment:
Therefore, the granularity is always 1/1000, right?

Reply:
discussed offline; will use math.MaxInt32 as the base to avoid issue that might arise from less than 1/1000